pandatool/src/bam/bamInfo.cxx

```diff
@@ -36,7 +38,7 @@ BamInfo() {
      "and Models native binary format--and describes their contents.");
 
   clear_runlines();
-  add_runline("[opts] input.bam [input.bam ... ]");
+  add_runline("[-ls -t -g -ltp] input.bam [input.bam ... ]");
```

Comment:
Is this the style used by other Panda cli tools? It looks like if one option is specified, then they all need to be there. I think leaving it as `[opts] ` is fine